packages/react-core/src/components/MultipleFileUpload/examples/MultipleFileUploadBasic.tsx

```diff
@@ -142,7 +142,7 @@ export const MultipleFileUploadBasic: React.FunctionComponent = () => {
       />
       <Checkbox
         id="upload-should-fail-checkbox"
-        label="Force uploads to fail"
+        label="Demonstrate error state with helper text"
```

Comment:
I think it would be best to combine.. with just the "demonstrate" i would expect something to happen as soon as i click the checkbox.

`"Demonstrate error reporting by forcing uploads to fail."`